roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -7,14 +7,29 @@
     docker pull {{ openshift.node.node_image }}:{{ openshift_image_tag }}
   register: pull_result
   changed_when: "'Downloaded newer image' in pull_result.stdout"
-  when: openshift.common.is_containerized | bool
+  when: openshift.common.is_containerized | bool and not openshift.common.is_node_system_container | bool
```

Comment:
@giuseppe Do you want to do what @detiber suggested in this PR or would it make sense to do it later?